tangorb/dao/src/main/java/fr/esrf/TangoApi/events/EventConsumer.java

```diff
@@ -53,13 +53,13 @@
         implements TangoConst, IEventConsumer {
 
     protected static int subscribe_event_id = 0;
-    protected static Hashtable<String, EventChannelStruct>   channel_map        = new Hashtable<String, EventChannelStruct>();
-    protected static Hashtable<String, String>               device_channel_map = new Hashtable<String, String>();
-    protected static Hashtable<String, EventCallBackStruct>  event_callback_map = new Hashtable<String, EventCallBackStruct>();
-    protected static Hashtable<String, EventCallBackStruct>  failed_event_callback_map = new Hashtable<String, EventCallBackStruct>();
+    protected static Hashtable<String, EventChannelStruct>   channel_map        = new Hashtable<>();
+    protected static Hashtable<String, String>               device_channel_map = new Hashtable<>();
+    protected static Hashtable<String, EventCallBackStruct>  event_callback_map = new Hashtable<>();
+    protected static Hashtable<String, EventCallBackStruct>  failed_event_callback_map = new Hashtable<>();
 
     //  Alternate tango hosts
-    static ArrayList<String>  possibleTangoHosts = new ArrayList<String>();
+    static ArrayList<String>  possibleTangoHosts = new ArrayList<>();
```

Comment:
@Pascal-Verdier  thanks for the clarification. Static is evil nowdays: [Why static is evil](https://stackoverflow.com/questions/7026507/why-are-static-variables-considered-evil) :) For now lets leave it as it is. We may change this once we drop support of Notifd.